packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2653,8 +2653,64 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// painting anything, without making the child available for hit testing, and
 /// without taking any room in the parent.
 ///
+/// Difference in painting can be easily compared by the following sections of with and without Offstage.
+///
+/// ### Without Offstage
+///
+/// ```dart
+/// import 'package:flutter/material.dart';
+///
+/// void main() {
+///   runApp(MyApp());
+/// }
+///
+/// class MyApp extends StatelessWidget {
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return MaterialApp(
+///       home: Scaffold(
+///         body: Container(
+///           color: Colors.red,
+///           height: 120.0,
+///           width: 120.0,
+///         ),
+///       ),
+///     );
+///   }
+/// }
+/// ```
+///
+/// ### With Offstage
+///
+/// ```dart
+/// import 'package:flutter/material.dart';
+///
+/// void main() {
+///   runApp(MyApp());
+/// }
+///
+/// class MyApp extends StatelessWidget {
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return MaterialApp(
+///       home: Scaffold(
+///         body: Offstage(
+///           child: Container(
+///             color: Colors.red,
+///             height: 120.0,
+///             width: 120.0,
+///           ),
+///         ),
+///       ),
+///     );
+///   }
+/// }
```

Comment:
General feedback: The example doesn't seem to be super-interesting. If you just want to hide a widget, we have better widgets for that purpose (e.g. `Visibility`).

Reply:
Yes, I would also choose visibility. 
The changes were related to Offstage only as mentioned on the issue and using visibility was already mentioned in see also section so I refrained from discussing on that.

Reply:
In general, the examples should show good practice, though, and using Offstage just to hide something is usually not the right thing to do.